readme/BREAKING_CHANGES.markdown

```diff
@@ -923,4 +923,26 @@ This requires no code changes. When you run the `buildService` task, you must ta
 
 This was done to clarify the folder that `buildService` should target.
 
+---------------------------------------
+
+## Deprecate `liferay-ui:toggle-value` tag
+- **Date:** 2021-Nov-26
+- **JIRA Ticket:** [LPS-142287](https://issues.liferay.com/browse/LPS-142287)
+
+### What changed?
+
+`liferay-ui:toggle-value` tag was deprecated
+
+### Who is affected?
+
+Everyone who is using this tag
+
+### How should I update my code?
+
+There isn't a replacement.
+
+### Why was this change made?
+
+It's producing a side effect (potential persistence action in the backend) during render in GET method and it's no longer used anywhere on DXP.
+
```

Comment:
This isn't technically a breaking change right? Are we planning to remove this tag in this version?

Reply:
IDK the process of deprecation a thing on DXP, so I assumed it will be removed on the next version

Reply:
> Are we planning to remove this tag in this version?

Can we do this? If yes, it will be nice :)

Reply:
The normal process is to keep the tag deprecated for at least a version, but with the latest changes to the release process this may change ¯\_(ツ)_/¯.

 In any case AFAICT we don't need to add an entry in the breaking changes file until we removed the tag, otherwise there is no breaking change. So I think we either:
  1) remove the tag + add the entry to the breaking changes file
  2) just deprecate the tag

I think I'd go for option 2, but I'm quoting @markocikos in case he think we should just remove the tag :p

Reply:
I agree, we should deprecate it. It's an ancient tag, 14 years old :t-rex:  . It is also pretty generic-looking. The more specific the tag is, and the more recently we added it, the safer would be to remove it.

Reply:
So, I'll remove the entry on breaking changes file

Reply:
Removed! Re-running CI things...